doc/source/tutorial/integrate.rst

```diff
@@ -445,49 +445,66 @@ has an exact solution using the matrix exponential:
 
 However, in this case, :math:`\mathbf{A}\left(t\right)` and its integral do not commute.
 
-There are many optional inputs and outputs available when using odeint
+There are many optional inputs and outputs available when using `solve_ivp`
 which can help tune the solver. These additional inputs and outputs
 are not needed much of the time, however, and the three required input
 arguments and the output solution suffice. The required inputs are the
-function defining the derivative, *fprime*, the initial conditions
-vector, *y0*, and the time points to obtain a solution, *t*, (with
-the initial value point as the first element of this sequence).  The
-output to :obj:`odeint` is a matrix where each row contains the
-solution vector at each requested time point (thus, the initial
-conditions are given in the first output row).
-
-The following example illustrates the use of odeint including the
-usage of the *Dfun* option which allows the user to specify a gradient
-(with respect to :math:`\mathbf{y}` ) of the function,
-:math:`\mathbf{f}\left(\mathbf{y},t\right)`.
-
-    >>> from scipy.integrate import odeint
-    >>> from scipy.special import gamma, airy
-    >>> y1_0 = 1.0 / 3**(2.0/3.0) / gamma(2.0/3.0)
-    >>> y0_0 = -1.0 / 3**(1.0/3.0) / gamma(1.0/3.0)
-    >>> y0 = [y0_0, y1_0]
-    >>> def func(y, t):
-    ...     return [t*y[1],y[0]]
-    ...
-
-    >>> def gradient(y, t):
-    ...     return [[0,t], [1,0]]
-    ...
-
-    >>> x = np.arange(0, 4.0, 0.01)
-    >>> t = x
-    >>> ychk = airy(x)[0]
-    >>> y = odeint(func, y0, t)
-    >>> y2 = odeint(func, y0, t, Dfun=gradient)
-
-    >>> ychk[:36:6]
-    array([0.355028, 0.339511, 0.324068, 0.308763, 0.293658, 0.278806])
-
-    >>> y[:36:6,1]
-    array([0.355028, 0.339511, 0.324067, 0.308763, 0.293658, 0.278806])
+function defining the derivative, *fprime*, the time span `[t_start, t_end]`
+and the initial conditions vector, *y0*. The output to :obj:`solve_ivp` is 
+a matrix where each row contains the solution vector for the given time 
+intervall. The initial conditions are therefore given in the first output row.
```

Comment:
```suggestion
function defining the derivative, *fprime*, the time span `[t_start, t_end]`
and the initial conditions vector, *y0*. :obj:`solve_ivp` outputs an object
whose *y* field is an array with consecutive solution values as columns.
The initial conditions are therefore given in the first output column.
```

Re: rows vs columns, this at least seems to be what https://docs.scipy.org/doc/scipy/reference/generated/scipy.integrate.solve_ivp.html currently suggests!